lms/templates/view.html

```diff
@@ -137,7 +137,7 @@ <h2>{{ _('הערות בודק') }}</h2>
 <script src="{{ url_for('static', filename='prism.js') }}"></script>
 {% if not shared_url %}
 <script src="{{ url_for('static', filename='comments.js') }}"></script>
-{%- if is_manager or config.USERS_COMMENTS %}
+{%- if is_manager or config.USERS_COMMENTS and solution['checker'] %}
```

Comment:
I think we should allow the user to add comments even if the manager still hasn't check the solution